contracts/package.json

```diff
@@ -19,7 +19,7 @@
   },
   "scripts": {
     "prepublishOnly": "hardhat clean && forge clean && hardhat compile && yarn build:forge:yul",
-    "coverage": "forge coverage --report lcov --ir-minimum && lcov --remove lcov.info 'node_modules/*' 'test/*' 'script/*' 'src/test-helpers/*' 'challenge/*' --ignore-errors unused -o lcov.info && genhtml lcov.info --branch-coverage --output-dir coverage",
```

Comment:
why?

Reply:
i was getting a stack too deep with how it was before, then moved some stuff around in test files to fix that, now ir-minimum isn't really necessary as far as i can tell

Reply:
adding it back gives stack problems again :upside_down_face: